assets/i18n/qqq.json

```diff
@@ -105,12 +105,19 @@
 	"I18N_CLASSROOM_CALLOUT_HEADING_1": "The header displayed in the callout to the classroom page which is present in the library page.",
 	"I18N_CLASSROOM_CALLOUT_HEADING_2": "The subheading for the callout to the classroom page which is present in the library page.",
 	"I18N_CLASSROOM_CALLOUT_PARAGRAPH_1": "Text for description of the callout to the classroom page which is present in the library page.",
+	"I18N_CLASSROOM_MATH_TITLE": "The text that is displayed on the classroom page - Text shown as classroom name.",
 	"I18N_CLASSROOM_PAGE_COMING_SOON": "The text displayed on a topic card in the classroom page when it is not published and inaccessible to the learner.",
 	"I18N_CLASSROOM_PAGE_COURSE_DETAILS": "The header displayed above the course details section in the classroom page.",
 	"I18N_CLASSROOM_PAGE_HEADING": "The header displayed above the classroom name in the classroom page.",
 	"I18N_CLASSROOM_PAGE_SEARCH_BAR_HEADING": "The heading for the search bar at the bottom of the classroom page.",
 	"I18N_CLASSROOM_PAGE_SEARCH_BAR_SUBHEADING": "The subheading for the search bar at the bottom of the classroom page.",
 	"I18N_CLASSROOM_PAGE_TOPICS_COVERED": "The header displayed above the topics list section in the classroom page.",
+	"I18N_COLLECTION_PLAYER_PAGE_BEGIN": "Part of title text displayed on classroom page - Text is shown when explorations are not started in a collection.",
```

Comment:
I'm not sure what "title text" is. Could you elaborate (here and below)?

If it's the title text for an image, explain what the image is.

Reply:
Done!